### DISCUSSION TITLE
Airdrop Issue

### DISCUSSION THREAD

**Discussion by lehoaivsybil** [zkSync-Community-Hub/zksync-developers#590] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

I saw have **Smart Contract Wallets** Multipliers in ZK airdrop

But seem **Smart Contract Wallets** on this Dapp not working on airdrop https://app.parallel.fi/zksync_era/apps/paraspace/credit

But **Smart Contract Wallets** on HOLDSTATION wallet is Eligibile. WHY?
Please check again and allocate some ZK for these **Smart Contract Wallets** as ZKNATION docs

**Comment by lehoaivsybil** [comment]
My EOA wallet is Eligibile for main ZK airdrop. But my **Smart Contract Wallets** which is created from https://app.parallel.fi/zksync_era/apps/paraspace/credit and have tx but not receive anything and my EOA wallet is **not Eligibile for Smart Contract Wallets Multipliers** Please check again

**Comment by 0xsisyfos** [comment]
Hello,

Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.

**Reply by lehoaivsybil** [reply]
But the problem people used https://app.parallel.fi/zksync_era/apps/paraspace/credit to create Smart Contract Wallets won't be count as a Multiplier. Smart Contract Wallets is created from HOLDstation wallet counted. WHY HAVE THIS DIFFERENT? Please help me forward that to your team to check

**Comment by lehoaivsybil** [comment]
But the problem people used https://app.parallel.fi/zksync_era/apps/paraspace/credit to create **Smart Contract Wallets** won't be count as a **Multiplier**. **Smart Contract Wallets** is created from HOLDstation wallet counted. WHY HAVE THIS DIFFERENT? Please help me forward that to your team to check
